e2e/test-data/secrets.md

```
@@ -0,0 +1,40 @@
+      // You can add other user roles here as needed
+      //
+      // memberUser: {
+      //   email: "member@example.com",
+      //   password: "anotherPassword"
+      // }
+    };
+    ```
+
+3.  **Verify `.gitignore`**: Ensure that the `secrets.ts` file path is included in your root `.gitignore` file to prevent it from ever being pushed to the repository. The entry should look like this:
+
+    ```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)